presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/IntegerStatisticsBuilder.java

```diff
@@ -54,22 +54,20 @@ public long getMaximum()
     private void addIntegerStatistics(long valueCount, IntegerStatistics value)
     {
         requireNonNull(value, "value is null");
-        requireNonNull(value.getMin(), "value.getMin() is null");
-        requireNonNull(value.getMax(), "value.getMax() is null");
```

Comment:
Boxing

Reply:
Should this be check value.hasMin() to be in parity with the old code.

Reply:
It's now done internally in the `getMinPrimitive`, so behavior stays the same.